app/graphql/types/customers/object.rb

```diff
@@ -65,6 +73,14 @@ def active_subscription_count
       def can_edit_attributes
         object.editable?
       end
+
+      def credit_notes_credits_available_count
+        object.credit_notes.where('credit_notes.credit_amount_cents > 0').count
```

Comment:
You're using the plural for `credits_available`but the singular for `credit_amount_cents`, not really consistent, right? 🤔

Reply:
hum, was about the count of them all so I put an `s`, still not convinced ? 😄

Reply:
true! my bad!